<?xml version="1.0" encoding="UTF-8"?><testrun name="UtilityTestSuite" project="unit-testing" tests="6" started="6" failures="1" errors="0" ignored="0">
  <testsuite name="com.psl.test.UtilityTestSuite" time="0.01">
    <testsuite name="com.psl.test.UtilityTest" time="0.008">
      <testcase name="testFactorial_1" classname="com.psl.test.UtilityTest" time="0.006">
        <failure>java.lang.AssertionError: Expected exception: java.sql.SQLException&#13;
	at org.junit.internal.runners.statements.ExpectException.evaluate(ExpectException.java:32)&#13;
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:26)&#13;
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:27)&#13;
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:271)&#13;
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:70)&#13;
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:50)&#13;
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:238)&#13;
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:63)&#13;
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:236)&#13;
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:53)&#13;
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:229)&#13;
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:26)&#13;
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:27)&#13;
	at org.junit.runners.ParentRunner.run(ParentRunner.java:309)&#13;
	at org.junit.runners.Suite.runChild(Suite.java:127)&#13;
	at org.junit.runners.Suite.runChild(Suite.java:26)&#13;
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:238)&#13;
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:63)&#13;
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:236)&#13;
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:53)&#13;
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:229)&#13;
	at org.junit.runners.ParentRunner.run(ParentRunner.java:309)&#13;
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:50)&#13;
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:459)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:675)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:382)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:192)&#13;
&#13;
</failure>
      </testcase>
      <testcase name="testFactorial_2" classname="com.psl.test.UtilityTest" time="0.0"/>
      <testcase name="testFactorial_3" classname="com.psl.test.UtilityTest" time="0.001"/>
    </testsuite>
    <testsuite name="com.psl.test.ParameterizedTest" time="0.002">
      <testsuite name="[0]" time="0.001">
        <testcase name="testFactorial[0]" classname="com.psl.test.ParameterizedTest" time="0.001"/>
      </testsuite>
      <testsuite name="[1]" time="0.0">
        <testcase name="testFactorial[1]" classname="com.psl.test.ParameterizedTest" time="0.0"/>
      </testsuite>
      <testsuite name="[2]" time="0.001">
        <testcase name="testFactorial[2]" classname="com.psl.test.ParameterizedTest" time="0.001"/>
      </testsuite>
    </testsuite>
  </testsuite>
</testrun>
